shell/platform/linux/fl_platform_views_plugin.cc

```diff
@@ -0,0 +1,310 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/fl_platform_views_plugin.h"
+
+#include <gtk/gtk.h>
+#include <cstring>
+
+#include "flutter/shell/platform/linux/fl_gesture_helper.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_method_channel.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_standard_method_codec.h"
+
+static constexpr char kChannelName[] = "flutter/platform_views";
+static constexpr char kBadArgumentsError[] = "Bad Arguments";
+static constexpr char kCreateMethod[] = "create";
+static constexpr char kDisposeMethod[] = "dispose";
+static constexpr char kAcceptGestureMethod[] = "acceptGesture";
+static constexpr char kRejectGestureMethod[] = "rejectGesture";
+static constexpr char kEnterMethod[] = "enter";
+static constexpr char kExitMethod[] = "exit";
+
+struct _FlPlatformViewsPlugin {
+  GObject parent_instance;
+
+  FlMethodChannel* channel;
+
+  FlGestureHelper* gesture_helper;
```

Comment:
I noticed `gesture_helper` isn't cleaned up in `dispose`. If the `_FlPlatformViewsPlugin` isn't supposed to take ownership of it, that might be worth noting here.

Reply:
Done.